src/scss/2022/components/_hero.scss

```diff
@@ -1,5 +1,6 @@
 @use "../tools/svg" as *;
 @use "../tools/color" as *;
+@use "../tools/media-queries" as *;
 
 .cmp-hero {
```

Comment:
Around 476px, the `.cmp-hero__intro` doesn't seem to respect the width of its container `.cmp-hero` and as a result the text bleeds to the edge of the window. 

<img width="475" alt="Screen Shot 2022-06-14 at 10 14 05 AM" src="https://user-images.githubusercontent.com/40482624/173599748-0bbc21b5-753c-4548-9918-60d2fc960e6f.png">